lib/msal-common/src/client/RefreshTokenClient.ts

```diff
@@ -62,20 +63,64 @@ export class RefreshTokenClient extends BaseClient {
         // We currently do not support silent flow for account === null use cases; This will be revisited for confidential flow usecases
         if (!request.account) {
             throw ClientAuthError.createNoAccountInSilentRequestError();
-        } 
+        }
 
-        const refreshToken = this.cacheManager.getRefreshTokenEntity(this.config.authOptions.clientId, request.account);
-        // no refresh Token
-        if (!refreshToken) {
-            throw ClientAuthError.createNoTokensFoundError();
+        // try checking if Family Refresh Token (FRT) is available
+        const familyRTResult = await this.acquireFamilyRefreshToken(request);
```

Comment:
I would suggest having `acquireFamilyRefreshToken` return the refresh token and then move line 77 down to outside the if/else

Reply:
Refactored